catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorKeyboardTest.java

```diff
@@ -110,52 +118,52 @@ public void basicMathOperatorKeysTest() {
 		onView(withId(R.id.formula_editor_keyboard_1)).perform(click());
 		onView(withId(R.id.formula_editor_keyboard_ok)).perform(click());
 
-		onView(withId(R.id.formula_editor_edit_field))
-				.check(matches(withText("( 1 ) + 1 - 1 × 1 ÷ 1 = 1 ")));
+		onView(withId(R.id.formula_editor_edit_field)).check(matches(withText("( 1 ) + 1 - 1 × 1 ÷ 1 = 1 ")));
 	}
 
 	@Category({Cat.AppUi.class, Level.Smoke.class})
 	@Test
 	public void enterStringTest() {
-		onView(withId(R.id.brick_set_variable_edit_text))
-				.perform(click());
-		onView(withId(R.id.formula_editor_keyboard_string))
-				.perform(click());
-
-		onView(withText(R.string.formula_editor_new_string_name))
-				.check(matches(isDisplayed()));
-		onView(withId(R.id.formula_editor_string_name_edit_text))
-				.perform(typeText("Foo"));
-		onView(withText(R.string.ok))
-				.perform(click());
-
-		onView(withId(R.id.formula_editor_keyboard_ok))
-				.perform(click());
-		onView(withId(R.id.formula_editor_edit_field))
-				.check(matches(withText("'Foo' ")));
-	}
+		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(1).checkShowsText(R.string.brick_change_size_by);
+		onBrickAtPosition(1).perform(click());
 
-	@After
-	public void tearDown() throws Exception {
+		onView(withId(R.id.formula_editor_keyboard_string)).perform(click());
+
+		onView(withText(R.string.formula_editor_new_string_name)).check(matches(isDisplayed()));
+		onView(withId(R.id.formula_editor_string_name_edit_text)).perform(typeText("Foo"));
+		onView(withText(R.string.ok)).perform(click());
+
+		onView(withId(R.id.formula_editor_keyboard_ok)).perform(click());
+		onView(withId(R.id.formula_editor_edit_field)).check(matches(withText("'Foo' ")));
 	}
 
-	public Project createProject(String projectName) {
-		Project project = new Project(null, projectName);
-		Sprite sprite = new Sprite("testSprite");
-		Script script = new StartScript();
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void deleteContentTest() {
+		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(1).checkShowsText(R.string.brick_change_size_by);
+		onBrickAtPosition(1).perform(click());
 
-		SetVariableBrick setVariableBrick = new SetVariableBrick();
-		DataContainer dataContainer = project.getDefaultScene().getDataContainer();
-		UserVariable userVariable = dataContainer.addProjectUserVariable("Global1");
-		setVariableBrick.setUserVariable(userVariable);
+		onView(withId(R.id.formula_editor_keyboard_1)).perform(click());
+		onView(withId(R.id.formula_editor_keyboard_1)).perform(click());
+		onView(withId(R.id.formula_editor_keyboard_delete)).perform(click());
+		onView(withId(R.id.formula_editor_edit_field)).check(matches(withText("1 ")));
+	}
 
-		script.addBrick(setVariableBrick);
-		sprite.addScript(script);
-		project.getDefaultScene().addSprite(sprite);
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void computeButtonTest() {
```

Comment:
please remove this test.
the compute dialog has a separate test class, and is actually more complex than you think testing wise and this will just cause trouble